integration_tests/pages/addMoreDetailsPage.ts

```diff
@@ -0,0 +1,25 @@
+import Page, { PageElement } from './page'
+
+export default class AddMoreDetailsPage extends Page {
+  h1 = (): PageElement => cy.get('h1')
+
+  description = (): PageElement => cy.get('[data-qa=alert-description]')
```

Comment:
You dont need to change this, but as an awareness thing we do have a `cy.getDataQa` method so you can cut down on the `[data-qa=...]` bits 🎉

Reply:
Oh yeah, forgot about that 👍